data/hai/unfettered 0 prologue.txt

```diff
@@ -515,3 +518,1020 @@ mission "Unfettered Jump Drive Trading"
+			label action
+			`	"It's pretty simple: you only need to go to <stopovers>, pick up <bunks> Hai and bring them to this planet. They will find their way from there." Seeing how everything here is in a... chaotic state, to say the least, one can only hope that to be true. The Hai you're fetching are hopefully aware of this already, or you may have to bring them back...`
+			choice
+				`	"I will get to it right away, then."`
+					accept
+				`	"I don't want to bring you Hai defectors. Sorry."`
+			`	He seems a bit shocked. "Defectors?? If you think that way then it is best you do not bring them indeed."`
+	on stopover
+		dialog `The small group of Hai are waiting for you in the spaceport as planned, and they seem happy to get on your ship. Now return to <origin> with them.`
+	to fail
+		has "unfettered: outcast"
+	on fail
+		dialog phrase "Essential Unfettered Mission Fail"
+	on visit
+		dialog phrase "generic missing stopover or passengers"
+	on complete
+		clear "unfettered: offered bring hai"
+		log `Helped some Hai citizens to join the Unfettered.`
+		payment 100000
+		dialog `The Unfettered thank you for helping these "True" Hai find their way home, and give you <payment>. After discussing with them, the Unfettered Chief comes back to address you, "If you want to help us again, the hospitals on Darkcloak are in grave need of resupply!" Checking back on the group, you can see they are doing fine and already finding a place to call home. They may not have much, but they seem not to require anything at all, other than to be here.`
+
+
+
+mission "<uhai> 1: Bring Medicine Notification"
+	landing
+	name `Need for medical supplies`
+	description `An Unfettered Chief told you to come help the hospitals on <destination> that desperately need supplies.`
+	source "Firelode"
+	destination "Darkcloak"
+	to offer
+		has "<uhai> 0: Bring Hai: done"
+
+
+
+mission "<uhai> 1: Bring Medicine"
+	landing
+	name `Bring medical supplies to <planet>`
+	description `The Unfettered hospitals on <origin> are in urgent need of resupply. Collect <tons> of medical supplies from <stopovers>, and bring it to the Unfettered on <destination> by <date> for <payment>.`
+	source "Darkcloak"
+	stopover "Stonebreak"
+	cargo "medicals" 50
+	blocked `You need <tons> of cargo space available to carry the medical supplies.`
+	deadline 9
+	to offer
+		has "<uhai> 0: Bring Hai: done"
+	substitutions
+		<prioritymessage> `Unfortunately, because it's in Hai, you can only guess at the meaning out of it, but it is clearly important. You can see the Hai word for "Unfettered" repeated several times, as well as numbers in the hundreds.`
```

Comment:
```suggestion
		<prioritymessage> `Unfortunately, because it's in Hai, you can only guess at the meaning of it, but it is clearly important. You can see the Hai word for "Unfettered" repeated several times, as well as numbers in the hundreds.`
```